apps/console/src/public/WEB-INF/web.xml

```diff
@@ -19,6 +19,11 @@
 
     <display-name>console</display-name>
 
+    <mime-mapping>
```

Comment:
Shall we add this to my account as well?

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/7618/commits/ade5fcc4b0b9187195bd237860d6afa33472bc70